src/Core/Nouns/FeminineNounPatternDetector.fs.fs

```diff
@@ -1,6 +1,6 @@
 ﻿module FeminineNounPatternDetector
 
-open Declensions
+open NounArticle
```

Comment:
Are the detectors called from the client code or are they used during the wiki scraping process?

Reply:
Only during scraping. Client does not any processing and only grabs stuff from the database.

Reply:
So this detector is not used from Server components? Also notice .fs.fs in the file name

Reply:
No, it's not used from there. Thanks, name fixed